src/components/login-modal.jsx

```diff
@@ -65,6 +82,16 @@ export default React.createClass({
               <li>Copy the new token and paste it in here!</li>
               <li><strong>Note:</strong> You may need to refresh the page when you click "Save"</li>
             </ol>
+            <h4>"GitHub Enterprise Endpoint"</h4>
+            <p>
+              If you need to set a custom API endpoint:<br/>
+              <BS.Input
+                type='text'
+                defaultValue={defaultRootURL}
+                placeholder='Enter GitHub API URL, e.g. https://github.example.com/api/v3'
+                ref='rootURL'
+              />
+            </p>
```

Comment:
Having said that, due to 94c043d I kept getting `token/rootURL has no method getValue defined`.